preview/Dockerfile

```diff
@@ -18,4 +18,5 @@ RUN apt-get update \
     && wget https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/Latest/dotnet-ubuntu-x64.latest.deb \
     && dpkg --force-depends -i dotnet-ubuntu-x64.latest.deb \
     && apt-get -f -y --no-remove install \
-    && rm dotnet-ubuntu-x64.latest.deb
+    && rm dotnet-ubuntu-x64.latest.deb \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
What do you think about also calling `apt-get clean`?  I'm wondering if the [Dockerfile best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get) forgot to include that in their example, because it seems like their explanation of the code implies it was included (or that something was used beyond deleting /var/lib/apt/lists/* at least) when it says "In addition, cleaning up the apt cache and removing /var/lib/apt/lists helps keep the image size down."